string message
---
float32 transmit_duration